community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+
+## Customized, performant and scalable administrative dashboard
+
+The other users they had to build for were the administrators of the conference. The WAC administrators had a few very specific requirements.
+
+- An extremely fast table to search, review and edit attendees information and status
+- Trends and analytics on sign ups over time, schools that registered and more
+- A content management system for the website and registration application
+
+In order to quickly find attendees to see if their registrations are complete, they ended up embedding [AG Grid](https://ag-grid.com/) into the admin app. SvelteKit wasn’t supported natively, so Samuel and his team had to create their own integration in order for the two to work well together. What you saw in this view and others depended entirely on the roles and permissions you had as an admin. Having this granular role based access control on the system was important as the number of users grew.
+
+![Attendee administrative table](/images/cs-wac-admin-table.png)
+
+In order to perform interactive analytics on the data itself, the team relied on [Xata aggregations](https://xata.io/docs/sdk/aggregate). This allowed them to chart the number of sign ups over time and segment based on schools and other profile attributes. The view of the data will help future conference administrators with outreach, promotion and further optimization.
+
+![Dashboard for attendee and speaker analytics](/images/cs-wac-admin-dashboard.png)
+
+Inline editing for the website was a really important feature in order to support real time modifications without providing full access to the frontend or administrative dashboard. Content administrators were able to make changes through this admin view, save and then automatically publish to the website. This was all handled through [Xata transactions](https://xata.io/docs/sdk/transaction). This CMS style editing was also used for user and speaker profiles, storing images with [file attachments](https://xata.io/docs/sdk/file-attachments). Having a single database that supported text, relationships and images was a really important part of the decision process when defining this tech stack to support multiple use cases.
+
+![Administrative CMS for website team](/images/cs-wac-admin-cms.png)
+
+Both the conference and administrative dashboard can be found in their [open source GitHub repository](https://github.com/worldaffairsconference/worldaffairscon.org).
```

Comment:
@Samuel-Martineau should we remove this sentence completely that references the OSS repo?

Reply:
No, this is still open source, but please link to https://github.com/worldaffairsconference/worldaffairscon.org/tree/wac2024